fuel-tx/src/tests/valid_cases/input.rs

```diff
@@ -495,7 +507,11 @@ fn message_message_coin() {
 
     let block_height = rng.gen();
     let err = tx
-        .check(block_height, &ConsensusParameters::standard())
+        .check(
```

Comment:
Thinking about this more. There are no `impl`s of `check` and only 4 `impl`s of `check_without_signature`. I'm not too worried about it. It would be a lot more work rn to create a new type--all the places we call those functions. And those places would still need to be changed in the future.

This also goes back to the work I did a while ago breaking down the params so functions only use what they need. As soon as we start grouping things together again it gets harder to uphold the Interface Segregation Principle.

Reply:
I'm pretty ambivalent about this one. I agree that it's a smell if we keep changing the interface, but I don't know if it's worth the work changing now.